ionoscloud/resource_nic_test.go

```diff
@@ -231,6 +237,12 @@ resource ` + constant.NicResource + ` "database_nic" {
   firewall_type = "INGRESS"
   ips = [ ionoscloud_ipblock.test_server.ips[0], ionoscloud_ipblock.test_server.ips[1] ]
   name = "%s"
+  flowlog {
+    name = "test_flowlog"
```

Comment:
@cristiGuranIonos not sure if this is required or not but I saw that we have multiple options that can be configured here, maybe we also want to have multiple tests for those options but I am not sure.

Reply:
solving this since the whole resource has been rewritten

Reply:
We have an update part below:
  flowlog {
    name = "test_flowlog_updated"
    action = "REJECTED"
    direction = "EGRESS"
    bucket = "` + constant.FlowlogBucketUpdated + `"
  }
This part takes care of update(deleting the nic also). We also have the last step of the test where the flowlog is missing which again forces re-creation of the nic and checks that it no longer exists.